src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -349,6 +349,7 @@ def __init__(
                 kernels.c.sess_id,
                 kernels.c.access_key,
                 kernels.c.domain_name,
+                kernels.c.user_uuid,
```

Comment:
Why is this line added though the column is not referenced below instantiation of `PendingSession`?

Reply:
I've forgot to remove while refactoring whole commit. :(